node/subsystem-types/src/messages.rs

```diff
@@ -85,6 +114,7 @@ impl BoundToRelayParent for CandidateBackingMessage {
 	fn relay_parent(&self) -> Hash {
 		match self {
 			Self::GetBackedCandidates(hash, _, _) => *hash,
+			Self::AdvertisementSecondingCheck { request, .. } => request.candidate_relay_parent,
```

Comment:
This BoundToRelayParent thing is outdated and the implementation here doesn't really make sense. But it has no effect since the removal of the jobs system.